src/azure-cli/azure/cli/command_modules/appconfig/_params.py

```diff
@@ -74,48 +75,50 @@ def load_arguments(self, _):
+        c.argument('separator', validator=validate_separator, help="Delimiter for flattening the json or yaml file to key-value pairs. Required for importing hierarchical structure. Not applicable for property files or feature flags. Supported values: '.', ',', ';', '-', '_', '__', '/', ':' ")
 
     with self.argument_context('appconfig kv import', arg_group='AppConfig') as c:
         c.argument('src_name', help='The name of the source App Configuration.')
-        c.argument('src_connection_string', validator=validate_connection_string, help="Connections of access key and endpoint of the source store. ")
-        c.argument('src_key', help='If no key specified, import all keys by default. Support star sign as filters, for instance abc* means keys with abc as prefix. Similarly, *abc and *abc* are also supported.')
+        c.argument('src_connection_string', validator=validate_connection_string, help="Combination of access key and endpoint of the source store. ")
+        c.argument('src_key', help='If no key specified, import all keys by default. Support star sign as filters, for instance abc* means keys with abc as prefix. Similarly, *abc and *abc* are also supported. Not applicable for feature flags.')
         c.argument('src_label', help="Only keys with this label in source AppConfig will be imported. If no label specified, import keys with null label by default.")
 
     with self.argument_context('appconfig kv import', arg_group='AppService') as c:
-        c.argument('appservice_account', validator=validate_appservice_name_or_id, help='ARM ID for AppService OR the name of the AppService, assuming the it is in the same subscription and resource group as the App Configuration. Required for AppService arguments')
+        c.argument('appservice_account', validator=validate_appservice_name_or_id, help='ARM ID for AppService OR the name of the AppService, assuming it is in the same subscription and resource group as the App Configuration. Required for AppService arguments')
 
     with self.argument_context('appconfig kv export') as c:
-        c.argument('label', help="Only keys with this label will be exported. If no label specified, export keys with null label by default.")
-        c.argument('prefix', help="Prefix to be trimed from keys.")
-        c.argument('key', help='If no key specified, return all keys by default. Support star sign as filters, for instance abc* means keys with abc as prefix. Similarly, *abc and *abc* are also supported.')
-        c.argument('destination', options_list=['--destination', '-d'], choices=['file', 'appconfig', 'appservice'], validator=validate_export, help="The destination of exporting.")
+        c.argument('label', help="Only keys and feature flags with this label will be exported. If no label specified, export keys and feature flags with null label by default.")
+        c.argument('prefix', help="Prefix to be trimmed from keys. Not applicable for feature flags.")
```

Comment:
There are many parameters which are not applicable for feature flags. Can you explain for me in brief? I am a little confused about the behavior when not applicable for feature flags.

Reply:
The idea is that specifying --prefix argument will not have any effect of feature flags. It will be ignored completely. Same for other parameters which are not applicable for feature flags.

---
In reply to: [355226236](https://github.com/Azure/azure-cli/pull/11467#discussion_r355226236) [](ancestors = 355226236)

Reply:
Thanks for explanation! But it will be better if you can make it more clear and easy understanding.

Reply:
done

---
In reply to: [356655044](https://github.com/Azure/azure-cli/pull/11467#discussion_r356655044) [](ancestors = 356655044)